src/code/z_collision_check.c

```diff
@@ -1271,6 +1250,10 @@ ColChkResetFunc sOCResetFuncs[] = {
     Collider_ResetQuadOC,   Collider_ResetSphereOC,
 };
 
+TriNorm D_801EE6C8;
+static s32 sBssPad;
```

Comment:
Hmm, just tested locally and this pad doesn't seem necessary?

Reply:
oh. Thats weird. I wonder why IDO decided to align this one. Removed